src/domain/kaivuilmoitus/BasicInfo.tsx

```diff
@@ -156,7 +164,7 @@ export default function BasicInfo({ johtoselvitysIds }: Readonly<Props>) {
         label={t('hakemus:labels:placementContracts')}
         tags={placementContracts ?? []}
         tagDeleteButtonAriaLabel={getTagDeleteButtonAriaLabel}
-        pattern="^SL\d{7}$"
+        pattern="^[sS][lL]\d{7}$"
```

Comment:
Why two different regex styles? For johtoselvitys `/^JS\d{7}$/i` and for sijoitussopimus `^[sS][lL]\d{7}$`? Could this regex be like for johtoselvitys: `{/^SL\d{7}$/i}`? Or the other way round?

Reply:
Text input which is used in sijoitussopimus only accepts string pattern so it's not possible to use the i flag. I can change johtoselvitys regex style to be similar to sijoitussopimus.

Reply:
Made the change.